src/jit/morph.cpp

```diff
@@ -8316,7 +8316,12 @@ GenTree* Compiler::fgMorphCall(GenTreeCall* call)
                                                                               : CORINFO_TAILCALL_NORMAL);
             if (pfnCopyArgs == nullptr)
             {
+#ifdef ALT_JIT
```

Comment:
I don't think we want `#ifdef ALT_JIT`; I think instead we want to use `info.compMatchedVM`

Reply:
Good point! I'll make the change. Do you think that this should be DEBUG-only?

Reply:
I don't think it needs to be DEBUG only; we use `compMatchedVM` in non-DEBUG contexts today.